.github/workflows/preview-closed.yml

```diff
@@ -0,0 +1,20 @@
+# file: .github/workflows/preview-closed.yaml
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  closed:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Context
+      uses: okteto/context@latest
+      with:
+        url: ${{ secrets.OKTETO_URL }}
+        token: ${{ secrets.OKTETO_TOKEN }}
+
+    - name: Destroy preview environment
+      uses: okteto/destroy-preview@latest
+      with:
+        name: pr-${{ github.event.number }}-bttf
```

Comment:
Did we want a different suffix or do we need to use your username because you configured everything?

Reply:
Yeah - we tried differently but Okteto won't accept a suffix other than the github username that owns the account